boards/opentitan/earlgrey-cw310/run.sh

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 
+BUILD_DIR="verilator_build/"
+
 if [[ "${VERILATOR}" == "yes" ]]; then
-	${OBJCOPY} --output-target=binary --strip-sections -S --remove-section .apps "${1}" binary
-	srec_cat binary \
+		if [ -d "$BUILD_DIR" ]; then
+			# Cleanup before we build again
+			printf "\n[CW-130: Verilator Tests]: Cleaning up verilator_build...\n\n"
+			rm -R "$BUILD_DIR"/*
+		else
+			printf "\n[CW-130: Verilator Tests]: Setting up verilator_build...\n\n"
+			mkdir "$BUILD_DIR"
+		fi
+	# Copy in and covert from cargo test output to binary
+	riscv64-linux-gnu-objcopy ${1} "$BUILD_DIR"/earlgrey-cw310-tests.elf
```

Comment:
You should use `${OBJCOPY}` here instaed

Reply:
${OBJCOPY} doesn't get set here, so we can't use it, without explicitly setting it i think.

Reply:
@alistair23 added support for the makefile to pass it into `run.sh`. This should keep things clean.